NuCivic/dkan
# Add tsv icon

## Description
• Add tsv icon to dkan-flaticons font, update css.
• Update nuboot_radix README.md to include font awesome in the icon credits. 

## QA Steps
1. Add a resource using a .tsv file
2. Confirm that you see a tsv icon.

![crime_data_for_the_ten_most_populous_cities_in_the_u_s____dkan](https://user-images.githubusercontent.com/314172/28125120-4100065a-66eb-11e7-9525-04cff18e7acb.png)
